CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
 <!-- SPDX-License-Identifier: CC0-1.0 -->
+<!-- doctest:ignore-file -->
```

Comment:
Perhaps some of the scripts in this file could be run, the only ones that shouldn't be run are those that invoke the tests because they would lead to infinite recursion.

Reply:
After evaluating all scripts in this file I've come to the following conclusion: all commands in this file should be executed in CI anyways, or are otherwise used during development, hence it does not make much sense to run them (again) as part of the tests.